Feature/queue 235

MohammedElazab opened the pull request:
Added country and phoneNumber to sign in form

![form](https://github.com/Brogammers/QMaster/assets/138368111/2abdd939-710c-4df0-83e7-f5dce3d5c5b1)
![form values](https://github.com/Brogammers/QMaster/assets/138368111/0dedd56b-54c0-49f2-aa08-3c2e748977ea)

CodeNKoffee commented:
@MohammedElazab please check if the Dates picker works after your adjustments. Also try commenting out the work you did and test the form and see if it works. If it does then the new input fields caused an issue.

MohammedElazab commented:
Made sure everything works fine and there is no problem in the field I added. The problem was that the token in authContext returns null for some reason. When I set a fixed token everything worked just fine. 
Also fixed account page bug.
![account](https://github.com/Brogammers/QMaster/assets/138368111/f1805674-1905-4a1c-807f-19830d4ec0fa)
